apps/extension/src/routes/page/page-layout.tsx

```diff
@@ -0,0 +1,40 @@
+import { Outlet } from 'react-router-dom';
+import { useStore } from '../../state';
+import { passwordSelector } from '../../state/password';
+import { useEffect } from 'react';
+import { PagePath } from './paths';
+import { usePageNav } from '../../utils/navigate';
+import { accountsSelector } from '../../state/accounts';
+
+const getLocationHash = (location: string, key: PagePath) =>
+  (location.replace('#', '') as PagePath) === key;
```

Comment:
What does this do? This needs a code comment also. This feels like a non-standard way of redirecting.